src/app/lib/presentation/diagnostic/expanded_diagnostic_page.dart

```diff
@@ -21,13 +22,14 @@ class _ExpandedDiagnosticPageState extends State<ExpandedDiagnosticPage>
     const DeviceInfoWidget(),
     EnvironmentDiagnosticWidget(),
     LoggerDiagnosticWidget(),
+    const MockingDiagnosticWidget(),
   ];
 
   int _selectedIndex = 0;
 
   @override
   void initState() {
-    _tabController = TabController(length: 4, vsync: this);
+    _tabController = TabController(length: 5, vsync: this);
```

Comment:
Can't we use `expandedDiagnosticWidgets.length` here? Or something similar?

Reply:
that's a good idea

Reply:
yes